### PR TITLE
Add CLA workflow

### PR DESCRIPTION
**Pull request by ivarpruijn**
Adds a GH Action workflow to run the [CLA Assistant bot](https://github.com/marketplace/actions/cla-assistant-lite) for new pull requests.

### CODE REVIEW

File: .github/workflows/cla.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+jobs:
+  CLAssistant:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        uses: cla-assistant/github-action@v2.1.3-beta
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          PERSONAL_ACCESS_TOKEN : ${{ secrets.CLA_ACCESS_TOKEN }}
```
Comment (ivarpruijn) [inline .github/workflows/cla.yml:16]: Added a new access token to the repository's secrets, with only `repo` rights.

File: .github/workflows/cla.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+        with:
+          path-to-signatures: '.github/cla/cla_signatures.json'
```
Comment (ivarpruijn) [inline .github/workflows/cla.yml:18]: Here all signatures are stored. Not sure doing it in the `.github` folder is the best place, but we also shouldn't create a new folder in the root for this.

File: .github/workflows/cla.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+          path-to-document: 'https://objectiv.io/cla/'
+          branch: 'main'
+          allowlist: dependabot,jansenbob,hendrik-obj,ivarpruijn,KathiaBarahona,borft,afroald,sdirosa,thijs-obj,thomashusken,jansentom,vincenthoogsteder
```
Comment (ivarpruijn) [inline .github/workflows/cla.yml:21]: These GH profiles can create PRs without having to sign the CLA first.

Comment (vincenthoogsteder) [reply .github/workflows/cla.yml:21]: also add vard?

Comment (ivarpruijn) [reply .github/workflows/cla.yml:21]: Good idea. Do you know her GH username?

**Comment by vincenthoogsteder** [comment]
vpapikya

---
Objectiv <https://objectiv.io/>
Co-founder & CEO
+31634190996
LinkedIn <https://www.linkedin.com/in/vhoogsteder/> / Twitter
<https://twitter.com/vincenthgstdr>

File: .github/workflows/cla.yml (hunk 3 above)
Comment (ivarpruijn) [reply .github/workflows/cla.yml]: Done

**Review by vincenthoogsteder — APPROVED** [review]
LGTM

**Pull request merged**